lrpar/cttests/src/quoting.test

```diff
@@ -0,0 +1,15 @@
+name: Test NoAction using the calculator grammar
+yacckind: Original(YaccOriginalActionKind::NoAction)
+grammar: |
+    %start S 
+    %%
+    S: '\' | '"' | '<' | '+' '🦀' ;
+
+lexer: |
+    %%
+    " '"'
+    \< '<'
+    \\ '\'
+    \+ '+'
+    🦀 '🦀'
+    [\t ]+ ;
```

Comment:
One thing I think would be nice is if we had an `input:` field, for giving example inputs that could run the compiled parser.